src/OrchardCore/Orchard.DisplayManagement/LocationExpander/ThemeAwareViewLocationExpanderProvider.cs

```diff
@@ -79,9 +79,10 @@ IEnumerable<string> PageViewLocations()
                     {
                         foreach (var theme in currentThemeAndBaseThemesOrdered)
                         {
-                            if (!context.PageName.StartsWith('/' + theme.Id + '/'))
+                            var themeViewsPath = "/" + theme.Extension.SubPath.Replace('\\', '/').Trim('/');
```

Comment:
I see that is so many places, can't we just normalize these at their source? Even if it means throwing an exception here if the path contains a `\` ?

Reply:
Yes, i did not dare to do it.

Razor also do many `NormalizePath()` because they work with `/` and we use in many place `Path.Combine()` which produces `\` on Windows ...

So, you're right, good to normalize some paths at the source when they are used for view paths, as it is the case for each extension `SubPath`. I will suggest something.